src/styles/index.scss

```diff
@@ -1,3 +1,2 @@
-body {
-  margin: 0;
-}
+@import 'variables';
+@import 'main';
```

Comment:
Ensure that the 'main' file exists in the same directory or provide the correct path if it is located elsewhere.